experiments/experiment_config_command_creator.py

```diff
@@ -26,22 +26,30 @@ class ExperimentConfigCommandCreator:
 
     @staticmethod
     def make_config(data_path, model_name, other_args):
-        mock_model_config = MockModelConfig("")
-        mock_model_config.start()
-
         checkpoint_dir = f"{data_path}/{model_name}"
 
         #yapf: disable
         args = [
             'model-analyzer', 'profile',
             '--profile-models', model_name,
             '--model-repository', data_path,
-            '--checkpoint-directory', checkpoint_dir,
-            '-f', 'path-to-config-file'
+            '--checkpoint-directory', checkpoint_dir
         ]
         args += other_args
 
-        yaml_content = convert_to_bytes("")
+        if '-f' not in args and '--config-file' not in args:
```

Comment:
Seems like there are two separate issues being solved in this story? What does this have to do with the failure to set metric weightings?

Reply:
Commit message may be a bit misleading. I don't think I fixed it, but merely just bypassed the issue. For some unknown reason in ArgumentParser in CLI class, specifying `--config-file` flag in command line causes an TypeError issue. This only happens for running experiments folder, so I'm bypassing it by reading the file and passing it to mock_config.

Reply:
I let Tim, weigh in on how clean we want to keep the experiments folder, but this seems hacky to me. Can you post the error message?

Reply:
I think this was to allow us to specify a custom yaml so we could test with custom constraints/objectives. The existing code just mocked out an empty yaml file, so this was his workaround.

Can you push the real fixes into main and this change into my side branch?

Reply:
Sure. I'll only leave the fixes for main on this PR.